src/chart/gauge/GaugeView.js

```diff
@@ -342,7 +342,7 @@ var GaugeView = ChartView.extend({
 
             pointer.useStyle(itemModel.getModel('itemStyle').getItemStyle());
 
-            if (pointer.style.fill === 'auto') {
+            if (pointer.style.fill === '#000') {
```

Comment:
If the user doesn't  specify `series.itemStyle.color`, there will be no problem.
But if the user specifies `#000` for `series.itemStyle.color`, according to current logic, the color of pointer won't be `#000` but an automatic color, for example `#2f4554`. That seems to be unexpected, pointer color should be `#000` in fact. 
So I think there may be missing a judgement about whether the user specifies `series.itemStyle.color`.
```js
var itemStyleModel = itemModel.getModel('itemStyle');

pointer.useStyle(itemStyleModel.getItemStyle());

if (pointer.style.fill === '#000' && !itemStyleModel.get('color')) {
    pointer.setStyle('fill', getColor(
        linearMap(data.get(valueDim, idx), valueExtent, [0, 1], true)
    ));
}
```

Reply:
Yes, you are right, I have added this judgement.